DOL.WHD.Section14c.Web/src/v4/dol-footer.component.ts

```diff
@@ -24,30 +24,43 @@ import { Component } from '@angular/core';
+          <ul class="usa-unstyled-list usa-width-one-fourth content-group" role="navigation">
             <li class="mb1"><strong>LABOR DEPARTMENT</strong></li>
-            <li><a href="#">Link 1</a></li>
-            <li><a href="#">Link 2</a></li>
-            <li><a href="#">Link 3</a></li>
-            <li><a href="#">Link 4</a></li>
+            <li><a href="http://www.dol.gov/dol/topic/Spanish-speakingTopic.htm">Español</a></li>
+            <li><a href="http://www.dol.gov/dol/siteindex.htm">A to Z Index</a></li>
+            <li><a href="http://www.dol.gov/dol/organization.htm">Agencies</a></li>
+            <li><a href="http://www.oig.dol.gov/">Office of Inspector General</a></li>
+            <li><a href="http://www.dol.gov/dol/contact/contact-phonekeypersonnel.htm">Leadership Team</a></li>
+            <li><a href="http://www.dol.gov/dol/contact/">Contact Us</a></li>
+            <li><a href="https://public.govdelivery.com/accounts/USDOL/subscriber/new?topic_id=USDOL_167" onclick="window.open('https://public.govdelivery.com/accounts/USDOL/subscriber/new?topic_id=USDOL_167&amp;pop=t', '', 'scrollbars=1,toolbar=0,menubar=0,resizable=1,width=800,height=420');return false;">Subscribe to the DOL Newsletter</a></li>
+            <li><a href="http://www.dol.gov/_sec/newsletter/">Read The DOL Newsletter</a></li>
+            <li><a href="http://www.dol.gov/easl/">Emergency Accountability Status Link</a></li>
           </ul>
-          <ul class="usa-unstyled-list usa-width-one-fourth content-group">
+          <ul class="usa-unstyled-list usa-width-one-fourth content-group" role="navigation">
             <li class="mb1"><strong>FEDERAL GOVERNMENT</strong></li>
-            <li><a href="#">Link 1</a></li>
-            <li><a href="#">Link 2</a></li>
-            <li><a href="#">Link 3</a></li>
-            <li><a href="#">Link 4</a></li>
+            <li><a href="http://www.dol.gov/cgi-bin/leave-dol.asp?exiturl=http://www.whitehouse.gov&amp;exitTitle=www.whitehouse.gov&amp;fedpage=yes">White House</a></li>
+            <li><a href="http://www.dol.gov/cgi-bin/leave-dol.asp?exiturl=https://www.healthcare.gov&amp;exitTitle=www.healthcare.gov&amp;fedpage=yes">Affordable Care Act</a></li>
+            <li><a href="http://www.dol.gov/opa/disaster-recovery.htm">Disaster Recovery Assistance</a></li>
+            <li><a href="http://www.dol.gov/cgi-bin/leave-dol.asp?exiturl=http://www.usa.gov&amp;exitTitle=www.usa.gov&amp;fedpage=yes">USA.gov</a></li>
+            <li><a href="http://www.dol.gov/dol/PlainWriting/">Plain Writing Act</a></li>
+            <li><a href="http://www.dol.gov/recovery/">Recovery Act</a></li>
+            <li><a href="http://www.dol.gov/oasam/programs/crc/NoFearResult.htm">No Fear Act</a></li>
+            <li><a href="http://www.dol.gov/cgi-bin/leave-dol.asp?exiturl=https://osc.gov/&amp;exitTitle=osc.gov&amp;fedpage=yes">U.S. Office of Special Counsel</a></li>
```

Comment:
These `&amp;` should be literal ampersands since they are part of a URL and not rendered content.  Otherwise, the exit page doesn't redirect properly.  (I'd love to have a whole other conversation about exit pages, but that's not relevant for this...  😬).

The affected links are to whitehouse.gov, healthcare.gov, usa.gov, and osc.gov.

Reply:
Fixed and pushed to the branch again.